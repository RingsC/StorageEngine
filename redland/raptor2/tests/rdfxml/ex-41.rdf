<?xml version="1.0"?>
<rdf:RDF xmlns:ex="http://example.org/ns#"
         xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">

  <!-- Check rdf:datatype attribute on property elements -->

  <rdf:Description rdf:about="http://example.org/node">
    <rdf:li rdf:datatype="http://example.org/datatype1">val1</rdf:li>
    <ex:prop2 rdf:datatype="http://example.org/datatype2">val2</ex:prop2>
    <ex:prop3 rdf:datatype="http://example.org/datatype3" xml:lang="de">val3</ex:prop3>
  </rdf:Description>

</rdf:RDF>
