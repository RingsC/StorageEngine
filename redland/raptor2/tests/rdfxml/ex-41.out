<http://example.org/node> <http://www.w3.org/1999/02/22-rdf-syntax-ns#_1> "val1"^^<http://example.org/datatype1> .
<http://example.org/node> <http://example.org/ns#prop2> "val2"^^<http://example.org/datatype2> .
<http://example.org/node> <http://example.org/ns#prop3> "val3"^^<http://example.org/datatype3> .
